src/compat/db_json.cpp

```diff
@@ -2871,6 +2884,23 @@ db_json_pretty_func (const JSON_DOC &doc, char *&result_str)
   result_str = db_private_strdup (NULL, json_pretty_writer.ToString ().c_str ());
 }
 
+std::string
+db_json_json_string_as_utf8 (std::string raw_json_string)
```

Comment:
We need to provide source encoding of input string.
My understanding is that JSON assumes input is utf-8; we cannot (or is difficult) to provide explicit input enconding, so the alternative would be to convert using CUBRID internal functions from DB_VALUE's encoding to utf-8, and then forward this new value to JSON.
This could be another issue/PR.